README.md

```diff
@@ -1,20 +1,80 @@
-# swagno: _no annotations, no files, no command_
+# Swagno: _Simplified Swagger Documentation Creation_
 
-<img src="https://user-images.githubusercontent.com/1047345/188009539-ea9d0106-979d-4f98-83a3-0d7df6969c9f.png" alt="Swagno" align="right" width="200"/>
+![Swagno Logo](https://user-images.githubusercontent.com/1047345/188009539-ea9d0106-979d-4f98-83a3-0d7df6969c9f.png "Swagno")
 
-Swagno is an approach to create Swagger Documentation 2.0 without any **annotation**, **exported file** or any **command** to run.
-You can declare your documentation details in code and get a json string to serve with a handler.
+Swagno offers a streamlined approach to create Swagger Documentation 2.0. With Swagno, you can declare documentation details directly in code, eliminating the need for annotations, exported files, or commands. This approach simplifies the process of generating a JSON string to serve with a handler.
```

Comment:
This is nice description but I like swagno's punchline: 
``` `no` annotation, `no` exported file,  `no` command to run ``` 😄
It describes the meaning of "swag`no`". Can you adapt this punchline in this description somehow 😄 Its not `must` btw you can skip this comment if you don't want to add.